build.gradle

```diff
@@ -40,6 +40,7 @@ android {
 
 dependencies {
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation "androidx.appcompat:appcompat:1.1.0-alpha04"
```

Comment:
Maybe rather go for a 1.0.x release instead of an alpha build when considering putting this PR as-is into a release.

Else it looks fine to me :+1:

Reply:
Yes, please use stable.

Reply:
Okay, I'll downgrade. I would like to mention that I experienced a couple of issues with version 1.0.0 in the news app.